### PR TITLE
chore: add app icon, app name and fix rotation to portrait

### PR DESCRIPTION
**Pull request by AnotherMedo**
_**QuickFix**_, being a utility app, should be primarily used in portrait mode for the best viewing experience. 

In order to further clean the look and feel of the app, an icon and name were added for the apk 🚀 

### Commit history:
`chore: add app icon, app name and fix rotation to portrait`:

- [x] Add an icon for the application in all available formats
- [x] Rename our app so that it shows _"QuickFix"_ when downloaded to a phone
- [x] Fix the orientation of the app to portrait mode

### CODE REVIEW

**Review by JitterKiller — CHANGES_REQUESTED** [review]
Small changes required for the name/icon, otherwise LGTM!

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -21,7 +21,8 @@
             android:name="com.arygm.quickfix.MainActivity"
             android:exported="true"
             android:label="@string/title_activity_main"
-            android:theme="@style/Theme.SampleApp">
+            android:theme="@style/Theme.SampleApp"
+            android:screenOrientation="sensorPortrait">
```
[the remark this answers is not in the document]
Comment (AnotherMedo) [reply app/src/main/AndroidManifest.xml:25]: The icon is directly taken from the `res` folder. By default it is a green icon with an android logo but I changed it to the QuickFix logo. The specific line is `android:icon="@mipmap/ic_launcher"`

[the remark this answers is not in the document]
Comment (JitterKiller) [reply app/src/main/AndroidManifest.xml:25]: Alright, sounds good

[the remark this answers is not in the document]
Comment (AnotherMedo) [reply app/src/main/AndroidManifest.xml]: > I think maybe it's better to use `portrait` orientation here as I don't really see the benefit or using reverse/sensor portrait here

done in `fix: change screenOrientation property of the app`

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
             android:exported="true"
             android:label="@string/title_activity_main"
             android:theme="@style/Theme.SampleApp"
-            android:screenOrientation="sensorPortrait">
+            android:screenOrientation="portrait">
```
Comment (JitterKiller) [inline app/src/main/AndroidManifest.xml:25]: Nice 👍

**Review by JitterKiller — APPROVED** [review]
LGTM!

**Pull request merged**